components/script/dom/audiotracklist.rs

```diff
@@ -70,29 +70,24 @@ impl AudioTrackList {
     }
 
     pub fn set_enabled(&self, idx: usize, value: bool) {
-        let track = match self.item(idx) {
-            Some(t) => t,
-            None => return,
-        };
-
-        // If the chosen tracks enabled status is the same as the new status, return early.
+        let track = self.item(idx).unwrap_or_else(|| return);
```

Comment:
This isn't the same as the previous code, because `||` here starts a closure and the return statement is returning from that closure rather than from `fn set_enabled`.